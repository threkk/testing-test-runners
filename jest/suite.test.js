import { describe, it, test, expect } from '@jest/globals'
import assert from 'node:assert'

describe('suite name', () => {
  it('foo', () => {
    assert.equal(Math.sqrt(4), 2)
  })

  it('bar', () => {
    expect(1 + 1).toEqual(2)
  })

  it('snapshot', () => {
    expect({ foo: 'bar' }).toMatchSnapshot()
  })
})

test.skip('this test will be skipped', () => {
  expect(true).toEqual(false)
})

test.todo('this test is a todo')
