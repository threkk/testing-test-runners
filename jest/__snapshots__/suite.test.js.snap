// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`suite name snapshot 1`] = `
Object {
  "foo": "bar",
}
`;
